www/js/index.js

```diff
@@ -71,11 +71,14 @@ openSonect: function() {
         uniqueIdentifier: "IBAN_1",
         name: "My Bank",
         detailDescription: "Balance: 20CHF",
+        funds: 20.0,
```

Comment:
I've added funds here to use internal check of balance 'cause we can't do it inside the app with callback from outer app.